docs/software/backpack-telemetry.md

```diff
@@ -0,0 +1,70 @@
+---
+template: main.html
+description: ExpressLRS Backpack can forward telemetry data to third party systems via ESPNOW.
+---
+
+![Software Banner](https://raw.githubusercontent.com/ExpressLRS/ExpressLRS-Hardware/master/img/software.png)
+
+## Introduction
+Version 3.4+ of ELRS introduces a new option in the Backpack section of the ELRS LUA script called "Telemetry." This feature allows a Backpack equipped ELRS Transmitter to forward CRSF telemetry frames to any peer that is capable of receiving ESPNOW messages. These frames can be consumed by any Backpack receivers that are bound with your bind phrase.
+
+<figure markdown>
+![Sentinel Integration](../assets/images/backpack-telemetry.png)
+<figcaption>Sentinel Integration</figcaption>
+</figure>
+
+This capability is particularly useful for devices like the Sentinel Antenna Tracker, which can subscribe to ESPNOW telemetry packets and read GPS data from the telemetry feed. See https://www.virtualpilot.co.uk/index.php?route=product/product&product_id=59
```

Comment:
I suggest using
`[Sentinel Antenna Tracker](https://www.virtualpilot.co.uk/index.php?route=product/product&product_id=59)`

(spellcheck is triggering errors with virtualpilot, uk & php)